test_common/harness/parseParameters.cpp

```diff
@@ -172,6 +181,11 @@ int parseCustomParam (int argc, const char *argv[], const char *ignore)
         return -1;
     }
 
+    if (gExecutionMode == kGenerate && (gCompilationCacheMode != kCacheModeOverwrite || gCompilationMode == kOnline))
```

Comment:
Why in online mode we cannot generate .cl ?

Reply:
We can add this to online compilation  mode as well if this is useful.
I'll add it to the to-do list

Reply:
I think this will help quick debugging Thanks